siliconcompiler/tools/vivado/route.py

```diff
@@ -0,0 +1,17 @@
+from siliconcompiler.tools import vivado
+
+def setup(chip):
+    '''Performs routing.'''
+    tool = vivado.tool
```

Comment:
couldn't you just import this as `from sc.tools.vivado import tool`

Reply:
will do, although it's kind of a wash line count wise (I don't like the implicit import of post_process, so I still want to keep the first namespaced import)